doc/userguide/upgrade.rst

```diff
@@ -30,6 +30,12 @@ by the ones Suricata supplies.
 Major updates include new features, new default settings and often also
 remove features.
 
+Upgrading 5.0 to 6.0
+--------------------
+
+Removals
+~~~~~~~~
+- File-store v1 has been removed. If using file extraction, the file-store configuration will need to be updated to version 2.
```

Comment:
I think we should link from here to the "Update File-store v1 Configuration to V2" chapter.